src/components/pages/docs.js

```diff
@@ -206,6 +207,22 @@ class Docs extends React.Component {
       : <span>{doc.updatedBy.username}</span>;
   }
 
+  renderOrigin(_value, doc) {
+    const { t } = this.props;
+    const origin = doc.origin || '';
+    let translatedOrigin = origin;
+
+    if (origin === DOCUMENT_ORIGIN.internal) {
+      translatedOrigin = t('originInternal');
+    }
+    if (origin.startsWith(DOCUMENT_ORIGIN.external)) {
+      const nameOfOrigin = origin.split('/')[1];
```

Comment:
You could also split on DOCUMENT_ORIGIN.external and not rely on the "/"being there. If you take the split[1] the "/" should be there already if we didn't misconfigure it (in which case this doesn't work either')

Reply:
maybe `origin.match(/^external\/(.+)$/)[1]`? Then the thing after "external/" can even contain another "/" ;-)

Reply:
I've split it by `DOCUMENT_ORIGIN.external` as Lau suggested, I find this is the simplest less error prone way. We can put `external` followed by whatever we want and only the `external` text will get translated

Reply:
Sure (unless the origin name contains "external") :laughing:

Reply:
Well, we get put in those names, I hope we don't do that. I'm not really sure why we really need to translate this value anyway, it seems a bit of overkill :)

Reply:
I just put this comment everywhere someone uses `string.split` when they actually want to remove a prefix, can't help, I find it the wrong function, sorry ;-)